### ISSUE TITLE
[Addon] Private relay button is not correctly displayed while using Norton Password Manager

### ISSUE THREAD

**Issue by sv-hyacoub** [mozilla/fx-private-relay#352]
### Affected versions

Firefox 76.0.1
Private Relay add-on version 1.4.0

### Affected platforms

Windows 10
Mac OS X 10.14
Ubuntu 18.04

### Steps to reproduce:

1. Launch Firefox.
2. Install the Private Relay add-on.
3. Install Norton Password Manager.
3. Go to reddit.com and click on sign up.

### Expected behavior
Private relay button should be correctly displayed while using Norton Password Manager.

### Actual behavior
Private relay button is not correctly displayed while using Norton Password Manager.
![Screenshot 2020-05-25 at 14 56 32](https://user-images.githubusercontent.com/15101135/82814705-5074a880-9ea0-11ea-8c68-863036d30a00.png)

**Comment by groovecoder** [comment]
Good catch, thanks! Exactly the kind of addon compat issue we want to find.
